workflow_support_scripts/convert_curve_files.py

```diff
@@ -0,0 +1,92 @@
+"""
+Convert RHFE sensor data into the correct format for ISIS
+"""
+
+import os
+from io import open
+import argparse
+from datetime import date
+import json
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+ORIGINAL_FILE_EXTENSION = ".curve"
+OUTPUT_FILE_EXTENSION = ".txt"
+isis_calibration = {"ISIS calibration": {"sensor_type": "RhFe",
+                                         "format_version": "1",
+                                         "conversion_date": date.today(),
+                                         "column1_name": "Temp",
+                                         "column1_units": "K",
+                                         "column2_name": "Measurement",
+                                         "column2_units": "Ohms"}
+                    }
+
+
+def get_input_output_folders():
+    parser = argparse.ArgumentParser(description='Get the input and output folder')
+    parser.add_argument('-i', "--input", dest="input_folder", type=str, help='The input folder', required=True)
+    parser.add_argument('-o', "--output", dest="output_folder", type=str, help='The output folder', required=True)
+    args = parser.parse_args()
+    return os.path.join(dir_path, args.input_folder), os.path.join(dir_path, args.output_folder)
```

Comment:
Could we add a docstring and return annotation to this method please? Maybe something like this:

```
def get_input_output_folders() -> Tuple(str, str):
    """
    Get the input and output folders from the command line
    :return: input_folder, output_folder
    """
```

You will need to import Tuple (`from typing import Tuple`) for the return annotation to work if using my example.